lib/connect/ws.js

```diff
@@ -4,13 +4,16 @@ var websocket = require('websocket-stream'),
   _URL = require('url');
 
 function buildBuilder (client, opts) {
+  var wsOpt = {};
+  wsOpt['protocol'] = 'mqttv3.1';
+  if(opts.protocol == 'wss')
+      if(opts.hasOwnProperty('rejectUnauthorized'))
+          wsOpt['rejectUnauthorized'] = opts.rejectUnauthorized;
```

Comment:
I fear this might cause some issue in the browser. Can you please check?
Otherwise I would prefer to add it only if it is not a browser.